runtime/astar/src/lib.rs

```diff
@@ -148,7 +148,7 @@ pub const VERSION: RuntimeVersion = RuntimeVersion {
     spec_name: create_runtime_str!("astar"),
     impl_name: create_runtime_str!("astar"),
     authoring_version: 1,
-    spec_version: 80,
+    spec_version: 81,
```

Comment:
Is the plan to make a release immediately after this PR?

I'm asking since there are some PRs I want to add for the next release, but want to wait until your merge yours so we avoid problematic rebasing/merging.

Reply:
My bad, I had to bump them for try-runtime testing.
I'll revert them so that we can be flexible on release.

Reply:
It's not a problem, I don't mind 🙂 

Wrote it more for info sharing, that some other things should also go into the next release.

Reply:
I reverted the spec version back, so we can be flexible on release